DuckDuckGo/AutocompleteViewController.swift

```diff
@@ -183,16 +183,7 @@ class AutocompleteViewController: UIViewController {
         selectedItem = -1
         tableView.reloadData()
 
-        loader = SuggestionLoader(dataSource: self, urlFactory: { phrase in
-            guard let url = URL(trimmedAddressBarString: phrase),
-                  let scheme = url.scheme,
-                  scheme.description.hasPrefix("http"),
-                  url.isValid else {
-                return nil
-            }
-
-            return url
-        })
+        loader = SuggestionLoader(dataSource: self, urlFactory: URL.makeURL(fromSuggestionPhrase:))
```

Comment:
I thought to align iOS with macOS. I moved the URLFactory function into BSK/Suggestions and have iOS & macOS use the function without duplicating the code.